org.evuz.visualReader.json

```diff
@@ -0,0 +1,147 @@
+        }
+    },
+    "modules": [
+        {
+            "name": "nodejs",
+            "cleanup": [
+                "/include",
+                "/share",
+                "/app/lib/node_modules/npm/changelogs",
+                "/app/lib/node_modules/npm/doc",
+                "/app/lib/node_modules/npm/html",
+                "/app/lib/node_modules/npm/man",
+                "/app/lib/node_modules/npm/scripts",
+                "/app/share/doc",
+                "/app/share/man"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://nodejs.org/dist/v8.9.3/node-v8.9.3.tar.gz",
+                    "sha256": "9efec12ae408f4e92738d9e5ab3c3fe68a407b7830228b0548ce9eef54cf63f9"                    
+                }
+            ]
+        },
+        {
+            "name": "yarn",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /app/share/yarn",
+                "cp -a * /app/share/yarn/",
+                "ln -s /app/share/yarn/bin/yarn /app/bin/yarn",
+                "ln -s /app/share/yarn/bin/yarnpkg /app/bin/yarnpkg"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/yarnpkg/yarn/releases/download/v1.3.2/yarn-v1.3.2.tar.gz",
+                    "sha256": "6cfe82e530ef0837212f13e45c1565ba53f5199eec2527b85ecbcd88bf26821d"
+                }
+            ] 
+        },
+        {
+            "name": "VisualComicReader",
+            "build-options" : {
+                "env": {
+                    "electron_config_cache": "/run/build/VisualComicReader/electron-cache",
+                    "npm_config_nodedir": "/app"
+                }
+            },
+            "buildsystem": "simple",
+            "build-commands": [
+                "yarn --offline --cwd /run/build/VisualComicReader/main install",
+                "yarn --offline --cwd /run/build/VisualComicReader/main run build:react",
+                "mkdir -p /app/main /app/bin",
+                "cp -ra main/* /app/main/",
+                "install -Dm755 VisualComicReader /app/bin/",
+                "install -Dm644 icon-256x256.png /app/share/icons/hicolor/256x256/apps/org.evuz.visualReader.png",
+                "install -Dm644 icon-128x128.png /app/share/icons/hicolor/128x128/apps/org.evuz.visualReader.png",
+                "install -Dm644 icon-64x64.png /app/share/icons/hicolor/64x64/apps/org.evuz.visualReader.png",
+                "install -Dm644 icon-48x48.png /app/share/icons/hicolor/48x48/apps/org.evuz.visualReader.png",
+                "install -Dm644 icon-32x32.png /app/share/icons/hicolor/32x32/apps/org.evuz.visualReader.png",
+                "install -Dm644 icon-16x16.png /app/share/icons/hicolor/16x16/apps/org.evuz.visualReader.png",
+                "install -Dm644 org.evuz.visualReader.desktop /app/share/applications",
```

Comment:
This needs to be `install -Dm644 org.evuz.visualReader.desktop /app/share/applications/org.evuz.visualReader.desktop `

Reply:
@adityashah1212 do you know if you'll have time to get that fix in?